src/app/components/markets/MarketsInfo.tsx

```diff
@@ -0,0 +1,21 @@
+import React from "react";
+
+export function MarketsInfo() {
+  const marketPairs = 0;
+  const xrdVolume = 0;
+  const volumeChange = 0;
+
+  return (
+    <div className="flex justify-between items-start">
```

Comment:
```suggestion
    <div className="flexg-grow flex justify-between items-start">
```

The footer jumps up to the middle of the page. Can be fixed by adding `flex-grow` class to this div.

![Screenshot 2024-01-26 at 18 16 34](https://github.com/DeXter-on-Radix/website/assets/27793901/e594d080-d89d-43cb-aa27-6e6efb303548)